data/a.json

```
@@ -11284,6 +11290,7 @@
             {
                 "id": "cc1a78ef04",
                 "def": "something having the property of being analogous to something else",
+                "example": "That argument is an analog of the one we just proved wrong.",
```

https://grammarist.com/spelling/analog-analogue/#:~:text=The%20spellings%20are%20largely%20interchangeable,all%20senses%20of%20the%20word.

```suggestion
                "example": "That argument is an analogue of the one we just proved wrong.",
```

---

This example is within the definition of the word that uses that spelling. That is to say, it doesn't make sense to have an example with "analog" withing the definition of "analogue"